gapic/src/main/com/google/gapid/server/GapidClientGrpc.java

```diff
@@ -29,6 +31,7 @@
  * A {@link GapidClient} based on a gRPC service.
  */
 public class GapidClientGrpc implements GapidClient {
+  private static final PingRequest PING_REQUEST = Service.PingRequest.newBuilder().build();
```

Comment:
PingRequest.getDefaultInstance() does the same, so no need to create a static constant.